app/consumers/peatio/model/deposit_updated_consumer.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Peatio
+  module Model
+    class DepositUpdatedConsumer
+      def call(event)
+        record = event.dig(:record)
+        transaction = OpenStruct.new(
+          email: record[:email],
```

Comment:
How are we getting email from the event?

Peatio dosent send email in the event.